content/coronavirus_landing_page.yml

```
@@ -306,7 +307,7 @@ content:
       next_conference_text: The next live press conference will be shown here
     # set spaced_links to true to insert breaks in the list of links, for increased legibility
     spaced_links: true
-    ask_a_question_visible: false
+    ask_a_question_visible: true
     ask_a_question_text: Ask a question at the next press conference
```

```suggestion
    ask_a_question_text: Ask a question at a press conference
```

Sorry! I'm making this PR so much harder. I'd forgotten we changed the terminology slighter. I got this change quickly confirmed by Michael Elwin our content designer.